src/app/components/ThemeProvider/palette.ts

```diff
@@ -43,3 +43,51 @@ export const STONE = '#D5D0CD';
 export const STORM = '#404040';
 export const WEATHER_BLUE = '#067EB3';
 export const WHITE = '#FFFFFF';
```

Comment:
Do we still need the single exports?

Reply:
Yea I think so, the theme for each service imports from `palette` https://github.com/bbc/simorgh/blob/c6d4442cc53b042390e3e98421227af9619ab582/src/app/components/ThemeProvider/themes/afaanoromoo.ts#L1

Reply:
👍